extensions/amp-gwd-animation/0.1/amp-gwd-animation.js

```diff
@@ -99,6 +97,16 @@ export class GwdAnimation extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Returns the GWD pagedeck element if one exists in the document.
+   * @return {?Element}
+   * @private
+   */
+  getGwdPageDeck_() {
+    return this.getAmpDoc().getRootNode().querySelector(
```

Comment:
1. If the selector is ID based, is there a reason we need to specify `amp-carousel`?
2. If so, please use `scopedQuerySelector` from `src/dom.js`.

Reply:
Changed to use getElementById(). I originally specified amp-carousel in the selector to add a guard against user error (e.g., duplicate elements with the ID), but this is not necessary.